airflow/models/trigger.py

```diff
@@ -116,7 +116,15 @@ def _decrypt_kwargs(encrypted_kwargs: str) -> dict[str, Any]:
         from airflow.models.crypto import get_fernet
         from airflow.serialization.serialized_objects import BaseSerialization
 
-        decrypted_kwargs = json.loads(get_fernet().decrypt(encrypted_kwargs.encode("utf-8")).decode("utf-8"))
+        # We weren't able to encrypt the kwargs in all migration paths,
+        # so we need to handle the case where they are not encrypted.
+        # Triggers aren't long lasting, so we can skip encrypting them now.
+        if encrypted_kwargs.startswith("{"):
+            decrypted_kwargs = json.loads(encrypted_kwargs)
```

Comment:
I think it's unlikely 🤔 but is it possible the first letter after encryption is `{`.

Reply:
Maybe we can catch the exception and run the else block below in that case?

Reply:
The [result is base64](https://cryptography.io/en/latest/fernet/#cryptography.fernet.Fernet.encrypt), so no risk. But if you want to refactor to decrypt/catch instead, have at it 👍

Reply:
If there's no risk, I think it's better keep it as it is for now to avoid unnecessary logic 👍

Reply:
i kindof like EAFP better.... i guess i'm using that acronym now...